### PR TITLE
Cleanup planning scene geometry code

### PR DESCRIPTION
**Pull request by sea-bass**
This PR cleans up some of the repetitive code in the planning scene geometry transcription, plus some other small stuff here and there.

I could get the code to compile, but for some reason the `vcs import` part of the container build is failing... so I'm unable to test it out on the latest image build. Can you guys try this out @sjahr @kamiradi?

I get this error:

```
Could not clone repository 'https://github.com/sjahr/moveit_benchmark_resources':
fatal: destination path '.' already exists and is not an empty directory.
```

### CODE REVIEW

**Comment by sea-bass** [comment]
Maybe this is the issue. When I manually try to clone your fork of the repo @sjahr , I get this error:

```
Downloading moveit_benchmark_resources/databases/panda_benchmarks.sqlite (4.3 MB)
Error downloading object: moveit_benchmark_resources/databases/panda_benchmarks.sqlite (43e9f02): Smudge error: Error downloading moveit_benchmark_resources/databases/panda_benchmarks.sqlite (43e9f02577ef1eda71d0ac2e63c79522791f78c3ef69fc24814af3def90bf299): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
```

**Comment by sjahr** [comment]
Oh no ... How can we fix this? Do you know if it is bandwidth / month and it will resolve itself next month?

**Comment by sjahr** [comment]
I looked a bit into it and we might have a problem if it does not reset on a monthly basis because otherwise we would need to lfs storage. A short-term workaround could also be to add the test database to this repository

File: src/ktopt_planning_context.cpp [hunk 1]
```diff
@@ -243,76 +244,54 @@ void KTOptPlanningContext::transcribePlanningScene(const planning_scene::Plannin
...
-        {
-          RCLCPP_WARN(getLogger(), "Cone not supported in drake");
+          const auto object_ptr = std::dynamic_pointer_cast<const shapes::Cylinder>(shape);
+          shape_ptr = std::make_unique<Cylinder>(object_ptr->radius, object_ptr->length);
           break;
         }
         default:
         {
-          RCLCPP_WARN(getLogger(), "Shape TYPE conversation to drake is not implemented");
+          RCLCPP_WARN(getLogger(), "Unsupported shape for '" + object_name + "', ignoring in scene graph.");
```
Comment (sjahr) [inline src/ktopt_planning_context.cpp:275]:
```suggestion
          RCLCPP_WARN(getLogger(), "Unsupported shape for '%s', ignoring in scene graph.", shape_name.c_str());
```

**Review by sjahr — APPROVED** [review]
Test it and it worked once you've included my comment (otherwise I get a compile error)

**Comment by sea-bass** [comment]
According to https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage, the overall storage caps out at 1GB, but bandwidth is 1GB per month. So it does reset.

But yeah, on another personal project I used to have a few GIFs on LFS and in a period of high activity it exceeded my quota. So I just brought it out to regular storage.

How big are these databases? If it's on the order of 10s of MBs, might be worth doing the same here. But if it's 100s... not sure. Maybe providing an external download link?

**Review by kamiradi — COMMENTED** [review]
A bit too late but thumbs up from me :)

**Pull request merged**